### PR TITLE
Save data every couple of seconds

### PR DESCRIPTION
**Pull request by bbrk24**
For battery testing, I will make a small local change to circumvent the need for a radio command to start it:
```diff
-    if (radioUart.available()) {
+    if (true) {
       //The string returned from readStringUntil does not include the terminator
-      String command = radioUart.readStringUntil('\n');
+      String command = "start";
```

### CODE REVIEW

**Review by CClarke513 — DISMISSED** [review]
Fine by me!

**Comment by bbrk24** [comment]
Marking this as draft until I figure out the performance issue

File: sketch_oct9a.ino [hunk 1]
```diff
@@ -324,6 +343,7 @@ void setup() {
 
   Scheduler.startLoop(gps_loop);
   Scheduler.startLoop(radio_loop);
+  Scheduler.startLoop(save_loop);
```
Comment (DarthSethious1) [inline sketch_oct9a.ino]: I understand this is necessary, but I feel like we should closely watch/look at how many different loops we're starting on a single-core system. If we put the 'saveToSD' stuff in the save loop as another, would it be quicker?

File: sketch_oct9a.ino [hunk 2]
```diff
@@ -374,4 +379,12 @@ void radio_loop() {
   sendDataToRadio();
 }
 
+void save_loop() {
+  delay(2000);
+  // If the file was intentionally closed, don't re-open it.
```
Comment (DarthSethious1) [inline sketch_oct9a.ino]: good save

**Review by DarthSethious1 — APPROVED** [review]
I'd like my comments to at least be commented on before I merge, but this looks good

File: sketch_oct9a.ino (hunk 1 above)
Comment (bbrk24) [reply sketch_oct9a.ino]: Yes, I was able to make it a bit faster by merging it with the GPS loop, but still nowhere near as fast as it was before. It's possible that simply writing the data to the SD card is a bottleneck and we should be careful how/when we do it.

Comment (DarthSethious1) [reply sketch_oct9a.ino]: I think you're looking into that already, so keep messing around with it and try and find what's best

**Review by CClarke513 — APPROVED** [review]
All good by me - Riley got things down to less than 1 Hz difference

**Review by DarthSethious1 — APPROVED** [review]
Everything looked good to me. Awesome job at ID-ing the problem and fixing it all within like 36 hours. Both you, Riley, and Cam are doing very well.

**Pull request merged**